src/style.css

```diff
@@ -1,3 +1,96 @@
+:root {
+  --blue-color: rgba(0, 172, 220, 1);
+}
+
+html,
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: inherit;
+  font-family: Roboto, serif;
+  font-weight: 400;
+}
```

Comment:
The universal selector '*' is used here, which is not recommended for performance reasons. You're also setting styles for all elements, even if they don't require them. Please consider applying these styles only to the elements that need them.